README.md

```diff
@@ -97,6 +101,5 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Microverse team for guidance
+- Based on [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on Behance design.
```

Comment:
[OPTIONAL] Because you are using Nelon Saka design and indicating it in the Readme file, it would be good to stick as close as you can to the design, including colors.
- Your design:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/77190203/175138366-2f24e574-462e-4291-a4dc-ce5d70b03ee3.png">

- Original design:
<img width="138" alt="image" src="https://user-images.githubusercontent.com/77190203/175139449-215e7d33-53ab-45dc-995c-c6a91af7967b.png">